### ISSUE TITLE
Building of rnpgp.com failed

### ISSUE THREAD

**Issue by ronaldtse** [rnpgp/rnpgp.org#16]
Error:

```ruby
      Generating... 
423             Error: could not read file /home/travis/build/rnpgp/rnpgp.com/vendor/bundle/ruby/2.5.0/gems/jekyll-3.8.6/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb: Invalid date '<%= Time.now.strftime('%Y-%m-%d %H:%M:%S %z') %>': Document 'vendor/bundle/ruby/2.5.0/gems/jekyll-3.8.6/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the YAML front matter.
424             ERROR: YOUR SITE COULD NOT BE BUILT:
425                    ------------------------------------
426                    Invalid date '<%= Time.now.strftime('%Y-%m-%d %H:%M:%S %z') %>': Document 'vendor/bundle/ruby/2.5.0/gems/jekyll-3.8.6/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the YAML front matter.
427rake aborted!
428
```

**Comment by ronaldtse** [comment]
Fixed above error in 52bcb7181e9ef39fae01aa08af351b4bf9f33e82 but new error below.

```ruby
 Incremental build: disabled. Enable with --incremental
389      Generating... 
390jekyll 3.8.6 | Error:  git '--git-dir=/home/travis/build/rnpgp/rnpgp.com/parent-hub/.git' '--work-tree=/home/travis/build/rnpgp/rnpgp.com/parent-hub' fetch 'origin'  2>&1:remote: Invalid username or password.
391fatal: Authentication failed for 'https://github.com/rnpgp/open.ribose.com/'
392rake aborted!
393Command failed with status (1): [JEKYLL_ENV=production jekyll build --confi...]
394/home/travis/build/rnpgp/rnpgp.com/Rakefile:65:in `block (2 levels) in <top (required)>'
395/home/travis/build/rnpgp/rnpgp.com/vendor/bundle/ruby/2.5.0/gems/rake-12.3.3/exe/rake:27:in `<top (required)>'
396/home/travis/build/rnpgp/rnpgp.com/vendor/bundle/ruby/2.5.0/bin/ruby_executable_hooks:24:in `eval'
397/home/travis/build/rnpgp/rnpgp.com/vendor/bundle/ruby/2.5.0/bin/ruby_executable_hooks:24:in `<main>'
398Tasks: TOP => test:production => build:production
399(See full trace by running task with --trace)
400The command "bundle exec rake test:production && rake build:production" exited with 1.
401
402
403
404
```

**Comment by ribose-jeffreylau** [comment]
Closing as it seems to be fixed.
